e2e-tests/commands/insurance/check-policy-summary-list.js

```diff
@@ -34,140 +34,125 @@ const {
   ACCOUNT: { EMAIL, FIRST_NAME, LAST_NAME },
 } = INSURANCE_FIELD_IDS;
 
-const { CONTRACT_POLICY, EXPORT_VALUE } = FIELDS;
-
 const { POLICY_CONTACT } = application;
 
 const checkPolicySummaryList = ({
   [REQUESTED_START_DATE]: () => {
     const fieldId = REQUESTED_START_DATE;
-    const expectedKey = CONTRACT_POLICY[fieldId].SUMMARY.TITLE;
-
     const { day, month, year } = application.POLICY[fieldId];
     const timestamp = createTimestampFromNumbers(day, month, year);
     const expectedValue = formatDate(timestamp);
 
-    const expectedChangeLinkText = CONTRACT_POLICY[fieldId].SUMMARY.TITLE;
+    const { expectedKey, expectedChangeLinkText } = getSummaryListField(fieldId, FIELDS.CONTRACT_POLICY);
 
     cy.assertSummaryListRow(summaryList, fieldId, expectedKey, expectedValue, expectedChangeLinkText);
   },
   [POLICY_CURRENCY_CODE]: () => {
     const fieldId = POLICY_CURRENCY_CODE;
-    const expectedKey = CONTRACT_POLICY[fieldId].SUMMARY.TITLE;
 
-    const currency = CURRENCIES.find((c) => c.isoCode === application.POLICY[fieldId]);
+    const { expectedKey, expectedChangeLinkText } = getSummaryListField(fieldId, FIELDS.CONTRACT_POLICY);
```

Comment:
could we dereference CONTRACT_POLICY and the others at the top?

Reply:
Good shout, updated this - we can dereference all except for `BROKER` because there are name clashes 👍